docs/user-guide/configuration.md

```diff
@@ -0,0 +1,98 @@
+# Harbor Satellite Configuration Guide
+
+This guide provides a complete reference for all configuration options in Harbor Satellite, explaining both the structure and practical usage of the configuration system.
+
+## Configuration Overview
+
+Harbor Satellite uses a JSON configuration file (`config.json`) for its settings. The configuration is divided into two main sections:
```

Comment:
- config.json file has changed 
- there is no separate zot config 
- config is now in the state coming from upstream registry